tka/tailchonk.go

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package tka
+
+import (
+	"os"
+	"sync"
+)
+
+// Chonk implementations provide durable storage for AUMs and other
+// TKA state.
+//
+// All methods must be thread-safe.
+//
+// The name 'tailchonk' was coined by @catzkorn.
+type Chonk interface {
+	// GetAUM returns the AUM with the specified digest.
+	//
+	// If the AUM does not exist, then os.ErrNotExist is returned.
+	GetAUM(hash AUMHash) (AUM, error)
```

Comment:
"Getters" in go often drop the Get, if they can get away with it. Usually on methods you can. I think you can here, without creating confusion?

Reply:
Done.